src/plugins/chapters/index.js

```diff
@@ -85,23 +85,38 @@ const ChaptersPlugin = (function () {
     }
   };
 
+  ChaptersPlugin.prototype.getAutoChaptersFileUrl = function getAutoChaptersFileUrl() {
+    const cloudName = this.player.cloudinary.cloudinaryConfig().cloud_name;
+    const currentPublicId = this.player.cloudinary.currentPublicId();
+
+    if (!cloudName || !currentPublicId) {
+      return null;
+    }
+
+    return `https://res.cloudinary.com/${cloudName}/video/upload/v1/${currentPublicId}-chapters.vtt`;
```

Comment:
what is the v1 here? i might be wrong but i do not think you need it?

Reply:
its necessary to get latest version - docs: https://cloudinary.com/documentation/invalidate_cached_media_assets_on_the_cdn